src/features/login/ui/LoginForm.tsx

```diff
@@ -35,11 +55,20 @@ const LoginForm: FC<BoxProps> = props => {
             name="password"
             placeholder={t('auth:password')}
           />
+
+          {error && (
+            <ErrorMessage error={{ message: error.evaluatedMessage! }} />
+          )}
         </YStack>
 
-        <Button variant="light" alignSelf="center" onPress={submit}>
-          {t('login_form:login')}
-        </Button>
+        <ProgressButton
+          isLoading={isLoading}
+          onClick={submit}
+          text={t('login_form:login')}
+          variant="light"
+          buttonStyle={{ width: 160, alignSelf: 'center' }}
```

Comment:
You can avoid setting the width here. It should automatically be calculated based on the text length. Also "alignSelf" styles  should be set on the `ProgressButton` level